### PR TITLE
Use ArrayBlockingQueue instead of LinkedBlockingQueue

### PR DESCRIPTION
**Pull request by ShashwatArghode**
Changing LinkedBlockingQueue to ArrayBlockingQueue as it is more performant in our case.

### CODE REVIEW

File: presto-cli/src/main/java/io/prestosql/cli/OutputHandler.java [hunk 1]
```diff
@@ -20,10 +20,10 @@
 import java.io.IOException;
```
Comment (kokosing) [inline presto-cli/src/main/java/io/prestosql/cli/OutputHandler.java]: Update cmt msg to:
```
Use ArrayBlockingQueue instead of LinkedBlockingQueue
```

**Comment by tooptoop4** [comment]
can this be merged?

**Comment by kokosing** [comment]
> can this be merged?

@tooptoop4 Sure, but there is still one pending comment that is not addressed: https://github.com/prestosql/presto/pull/2231#discussion_r355605362

FYI: @ShashwatArghode ping

File: presto-cli/src/main/java/io/prestosql/cli/OutputHandler.java (hunk 1 above)
Comment (ShashwatArghode) [reply presto-cli/src/main/java/io/prestosql/cli/OutputHandler.java]: Done

**Comment by ShashwatArghode** [comment]
@kokosing, @tooptoop4: Sorry for the delay on my end, I have updated the commit message.

**Comment by kokosing** [comment]
Merged, thanks!

**Pull request merged**